package day5.exception;

public class Main {
 public static void main(String[] args) {
	 try {
		 int a=43/0;
 
	 }catch(Exception e) {
		 System.out.println(e);
	 }
 }
}
